packages/ts-codegen/README.md

```diff
@@ -1,3 +1,10 @@
-# @graphitation/graphql-codegen
+# @graphitation/ts-codegen
+
+Package is used for [supermassive](../cli/src/supermassive.ts) cli to generate typescript code from a GraphQL document.
```

Comment:
```suggestion
This package generates Typescript definitions based on GraphQL Schema Definition Language.
```

Reply:
Sure, changed in https://github.com/microsoft/graphitation/pull/498/commits/94e2723505bde4b69a80721205de5e802a76ea22